tests/unit_tests/ModuleBindingTests.cpp

```diff
@@ -118,7 +118,9 @@ endmodule
 
     // Check that the tree of children has been instantiated correctly
     const auto& leaf = instance->getChild<InstanceSymbol>(0).getChild<InstanceSymbol>(0);
-    CHECK(std::get<SVInt>(leaf.module->parameters[0]->value) == 4);
+    const auto& foo = leaf.module->scope->lookup(StringRef(std::string("foo")))->as<ParameterSymbol>();
```

Comment:
No need to wrap "foo" in std::string, it should work if you just pass that to StringRef. In fact it's an implicit conversion, so you should be able to just do lookup("foo",